Created a demo Page using HTML and CSS.

Yashasviny opened the pull request.

Yashasviny committed:
Created a demo Page using HTML and CSS.

Yashasviny committed:
Changes in TreeHouse page.

Yashasviny committed:
HTML page.

Yashasviny commented:
@anjanaNewput  Please review the changes.

anjanaNewput commented on htmlCssTasks/secondTask/zoror.css:
```diff
@@ -0,0 +1,172 @@
+.friend {
+  color: #c67676;
+  display: block;
+
```
Don't leave space after property in css.

anjanaNewput commented on htmlCssTasks/secondTask/zoror.html:
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="zoror.css"></link>
```
Why do you need to close link tag manually? any specific reason?

Yashasviny committed:
CSS Icons

Using Icons in Css with font-awesome library.

Yashasviny committed:
About Shadows,Animations and Transitions.

Explore about Shadows,Animations and Transitions.

Yashasviny committed:
About SCSS

Explore SCSS including some basic features like mixins, variables,
operators, extends, partials, nesting.

anjanaNewput commented on htmlCssTasks/sass/First_Example/content.html:
```diff
@@ -0,0 +1,11 @@
+<html>
+   <head>
+      <title> Import example of sass</title>
+      <link rel = "stylesheet" type = "text/css" href = "output.css"/>
```
Make sure, not to give space between attribute and its value.
rel = "stylesheet"  should be  rel="stylesheet"

Yashasviny replied:
Okay Ma'am

Yashasviny committed:
Tiles Structure.

Using HTML and SCSS combine to design a tile structure.
